src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_primitive_fusing.cpp

```diff
@@ -704,12 +705,16 @@ void prepare_primitive_fusing::fuse_simple_primitives(program &p) {
             if (_lo.get_optimization_attributes().use_onednn_impls) {
                 if (input.is_type<reshape>() || input.is_type<concatenation>())
                     return;
-
+                if (activation_func == cldnn::activation_func::relu_negative_slope &&
+                    (input.is_type<fully_connected>() || input.is_type<gemm>())) {
```

Comment:
this is a black-list way, but you said `prelu` is supported for conv only. Then shouldn't we use white-list for `conv`, instead?

Reply:
Only Conv is not an official, just my idea after reviewed the onednn code.
I think to use white-list is good after okay from onednn(they agree only conv can support it), or test it in our side.